src/components/Loader/PersonLink.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  selectedPerson: string;
+};
+
+export const PersonLink: FC<Props> = ({
+  person: {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  },
```

Comment:
Better to destructure the value inside the component, not in the props directly

Reply:
fixed